lms/djangoapps/course_structure_api/v0/views.py

```diff
@@ -62,6 +62,7 @@ def course_check(func):
         :param func: function to be wrapped
         :returns: the wrapped function
         """
+
```

Comment:
Don't make these changes (see whole file) unless you are fixing a quality issue in the automated tests.  Thanks.